lib/libfrr.c

```diff
@@ -1229,16 +1233,15 @@ void frr_fini(void)
 	/* frrmod_init -> nothing needed / hooks */
 	rcu_shutdown();
 
-	if (!debug_memstats_at_exit)
-		return;
-
-	have_leftovers = log_memstats(stderr, di->name);
+	/* also log memstats to stderr when stderr goes to a file*/
+	if (debug_memstats_at_exit || !isatty(2))
```

Comment:
Discussed this offline and consensus was that we should look into this later, and it might should offer option to specify path to memstat files that are generated when this option is enabled (instead of hard coded to /tmp)